include/swift/SIL/SILModule.h

```diff
@@ -88,6 +88,16 @@ class FixedSizeSlab : public llvm::ilist_node<FixedSizeSlab>,
   template<typename T> const T *dataFor() const { return (const T *)(&data[0]); }
 };
 
+/// A fixed size slab of memory, which can be allocated and freed by the
+/// SILModule at (basically) zero cost.
```

Comment:
Can you add a comment here explaining in a paragraph below why it is zero cost? Just that way people do not have to look around.

Reply:
I'll add a "See SILModule::allocSlab()", which has more info